nuxeo-arender-rendition-apb/templates/arender_converter_service.yml.j2

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: {{ app_name }}
+    apb_name: '{{ apb_name }}'
+    apb_plan_id: '{{ _apb_plan_id }}'
+    apb_service_class_id: '{{ _apb_service_class_id }}'
+    apb_service_instance_id: '{{ _apb_service_instance_id }}'
+  name: {{ arender_converter_service_name }}
+  namespace: {{ namespace }}
+spec:
+  type: ClusterIP
+  clusterIP: None
+  ports:
+    - name: 8761-tcp
+      port: 8761
+      protocol: TCP
+      targetPort: 8761
+  selector:
+    app: {{ app_name }}
+    role: rendition
```

Comment:
I would update that to `rendition-converter`. For every service, if you don't update that, each service will point to every DC that you defined. That's why we need a different role for each DC.